Library/Tracking/KSRAnalytics.swift

```diff
@@ -1347,7 +1346,12 @@ public final class KSRAnalytics {
   }
 
   public func trackLoginSubmitButtonClicked() {
-    self.track(event: ApprovedEvent.loginSubmitButtonClicked.rawValue, page: .login)
+    let props = contextProperties(ctaContext: .logInSubmit)
```

Comment:
The ticket requires `userProperties` and I'm not seeing those on the segment debugger.